source/game_sa/Coronas.cpp

```diff
@@ -156,9 +154,17 @@ void CCoronas::Update() {
     };
 
     if (currLook == LastCamLook) {
-
+        // NOTE: R* messed around with changing the variable here, maybe bChangeBrightnessImmediately wasn't bool after all?
+        bChangeBrightnessImmediately = false;
+    } else {
+        bChangeBrightnessImmediately = true;
+    }
+    LastCamLook = currLook;
```

Comment:
It's definitely an enum
```suggestion
    if (std::exchange(LastCamLook, currLook) == currLook)  {
        bChangeBrightnessImmediately  = bChangeBrightnessImmediately  <= 0
            ? 0 
            : bChangeBrightnessImmediately - 1;
    } else {
        bChangeBrightnessImmediately  = 3;
    }
```